tests/require.rs

```diff
@@ -708,25 +708,23 @@ fn issue_1158_conflicting_requirements_rev() {
 fn issue_1643_args_mutually_require_each_other() {
     use clap::*;
 
-    fn main() {
```

Comment:
This test is failing

Reply:
Wow, the test code haven't been being run! It have effectively been checking that it compiles, and that was all.